### ISSUE TITLE
systemctl restart netdata hanging at systemd-tty-ask-password-agent

### ISSUE THREAD

**Issue by 0xpr03** [netdata/netdata#11555]
##### Bug report summary
`systemctl restart netdata` takes many seconds with a child process "systemd-tty-ask-password-agent --watch"

##### OS / Environment
Debian 9

```
Linux 4.19.0-17-amd64 #1 SMP Debian 4.19.194-3 (2021-07-18) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 10 (buster)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="10"
/etc/os-release:VERSION="10 (buster)"
/etc/os-release:VERSION_CODENAME=buster
/etc/os-release:ID=debian
/etc/os-release:HOME_URL="https://www.debian.org/"
/etc/os-release:SUPPORT_URL="https://www.debian.org/support"
/etc/os-release:BUG_REPORT_URL="https://bugs.debian.org/"
```

##### Netdata version

 netdata v1.31.0-279-nightly

##### Installation method

bash script per github readme

##### Component Name

service control

##### Steps To Reproduce

1. install
2. systemctl restart netdata
3. watch it hang with htop showing the specified process

##### Expected behavior

No hang on that process over minutes

**Comment by ilyam8** [comment]
Hi, @0xpr03. 

Does `systemctl restart netdata` always hang? Or sometimes?

Having `systemd-tty-ask-password-agent` process is expected, [it always runs alongside](https://github.com/systemd/systemd/issues/9507#issuecomment-404078079) `systemctl` whenever invoked

<details>
  <summary>a screenshot</summary>

<img width="706" alt="Screenshot 2021-09-23 at 10 12 23" src="https://user-images.githubusercontent.com/22274335/134467654-405a1637-f100-46c8-b730-682ac4fd3db3.png">

</details>

**Comment by 0xpr03** [comment]
It always hangs, systemd also reports killing the shutdown process due to timeout.

**Comment by ilyam8** [comment]
Was it a clean install? Can you remove Netdata and install it again and see if the problem still exists.

**Comment by 0xpr03** [comment]
Yeah it's a clean install, was right after installation as I wanted it to bind only to 127.0.0.1

**Comment by ilyam8** [comment]
@0xpr03 I suspect the same problem as I get in https://github.com/netdata/netdata/issues/11615

Let's try to disable `ebpf.plugin` and see if it helps


```cmd
# netdata.conf
[plugins]
  ebpf = no
```
